Modules/_ctypes/callproc.c

```diff
@@ -1123,6 +1123,15 @@ PyObject *_ctypes_callproc(PPROC pProc,
                 goto cleanup; /* leaking ? */
             }
         }
+#if (defined(__x86_64__) && (defined(__MINGW64__) || defined(__CYGWIN__))) || \
+        defined(__aarch64__)
+        void *tmp;
```

Comment:
Just a little tidier. The reference (`tmp`) stays in the scope in which it's used.